### PR TITLE
Styling Tabs

### LINKED ISSUE #25: Stylize Vertical Tab component

### PR DESCRIPTION
**Pull request by oupadhyay**
Closes #25

### CODE REVIEW

File: packages/client/src/components/Tabs/mui_nested.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Box from '@material-ui/core/Box';
 import { useGatsbyNestedMenuStyles } from '@mui-treasury/styles/nestedMenu/gatsby';
```
Comment (mi-yu) [inline packages/client/src/components/Tabs/mui_nested.js:3]: If this is just an example, let's leave it out of this PR

File: packages/client/src/components/Tabs/index.js [hunk 2]
```diff
@@ -66,34 +103,32 @@ export default function VTabs(props) {
 
     return (
         <div className={classes.root}>
-            <Tabs
+            <ProcessTabs
                 /* sets horizontal or vertical tabs */
                 orientation="vertical"
                 /* changes type of tab feature if tabs > height*/
                 // variant="scrollable"
```
Comment (mi-yu) [inline packages/client/src/components/Tabs/index.js:110]: let's delete this comment

File: packages/client/src/components/Tabs/index.js [hunk 3]
```diff
@@ -66,34 +103,32 @@ export default function VTabs(props) {
...
-                            {text}
-                        </TabPanel>
-                    )
-                )
-            }
+                {labels.map((label, index) => (
+                    <Tab
+                        key={index}
+                        label={label}
+                        style={index === value ? color : defaultColor}
```
Comment (mi-yu) [inline packages/client/src/components/Tabs/index.js:121]: we actually don't need to track the `color` state, we can just do `index === value ? selected : defaultColor`

**Comment by mi-yu** [comment]
Seems good overall just some small comments

eventually we want to remove the ripple effect when you click on a tab item and also add the triangles, but we can do that in another pr

File: packages/client/src/components/Tabs/index.js (hunk 2 above)
Comment (oupadhyay) [reply packages/client/src/components/Tabs/index.js:110]: which comment?

Comment (mi-yu) [reply packages/client/src/components/Tabs/index.js:104]: `                // variant="scrollable"`

**Pull request merged**